src/constant/types.ts

```diff
@@ -35,6 +36,7 @@ export interface InitialProps {
   numberOfMonths?: number;
   initialMonthAndYear?: string;
   onRangeDateInScreen?: DatePickerWindowUpdated;
+  dayClasses?: (day: Dayjs) => string[];
```

Comment:
That's great, Colon, Thank you so much for your great feature. 

Before we are going to merge it, can we have a specific type for dayClasses and export it and use it in all places?

Reply:
Good shout. Will get that sorted at some point soon (for various values of "soon" 😛 )